help/en/html/doc/Technical/NetBeans.shtml

```diff
@@ -17,22 +17,22 @@
     <div id="mainContent">
       <!-- this was created by running the command `asciidoctorj -a toc NetBeans.adoc`
             and copying the contents of body element in the resultant NetBeans.html file -->
-      <!-- manual clean-up is required to make this shtml responsive -->
+      <!-- manual clean-up was required to make this shtml responsive -->
```

Comment:
If this cleanup is done, I suggest remove that comment.

Reply:
comment changed in #11584 now deleted.